app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -52,16 +52,22 @@ class AddDeadlineActivity : AppCompatActivity() {
 
     private val dateTimeExtractor = DateTimeExtractor { clockService.now().toLocalDate() }
     private val deadlineListViewModel: DeadlineListViewModel by viewModels()
+    private val groupViewModel: GroupViewModel by viewModels()
     private lateinit var textDate: TextView
     private lateinit var textTime: TextView
-    private lateinit var editText: TextView
     private lateinit var pdfTextView: TextView
     private lateinit var textTitle: TextView
     private lateinit var textDescription: TextView
 
     // Memorisation of which checkBox is selected for the notifications
     private val notificationSelected = booleanArrayOf(false, false, false, false)
-    private val nameCheckBox = arrayOf("1 hour", "5 hours", "1 day", "3 days")
+    private val nameNotifications = arrayOf("1 hour", "5 hours", "1 day", "3 days")
```

Comment:
I think this could be in the companion object (and therefore uppercase). And unless it is modified somewhere, which I don't think is the case, it would be better to make it a list.

Reply:
The companion object is true, but we need it in array in a function